packages/ui/src/views/repo/pull-request/compare/components/pull-request-compare-form.tsx

```diff
@@ -46,16 +126,82 @@ const PullRequestCompareForm = forwardRef<HTMLFormElement, PullRequestFormProps>
             size="md"
           />
 
-          <Textarea
-            id="description"
-            {...register('description')}
-            placeholder={t(
-              'views:pullRequests.compareChangesFormDescriptionPlaceholder',
-              'Add Pull Request description here.'
-            )}
-            label={t('views:pullRequests.compareChangesFormDescriptionLabel', 'Description')}
-            error={errors.description?.message?.toString()}
-          />
+          <div
+            className={cn('pb-5 pt-1.5 px-4 flex-1 bg-background-1 border border-borders-2 rounded-md', {
+              // 'border rounded-md': !inReplyMode || isEditMode,
+              // 'border-t': inReplyMode
+            })}
+          >
+            <Tabs variant="tabnav" defaultValue={TABS_KEYS.WRITE} value={activeTab} onValueChange={handleTabChange}>
+              <TabsList className="relative left-1/2 w-[calc(100%+32px)] -translate-x-1/2 px-4">
```

Comment:
Can we move `32px` to a css variable instead? 

You can add it to `apps/gitness/src/styles/styles.css`

```
:root {
  --tab-width: 32px;
}

.tab-width {
  width: var(--tab-width);
}
```

Reply:
fixed @vardanbansal-harness